types/time_test.go

```diff
@@ -1812,86 +1812,3 @@ func (s *testTimeSuite) TestFromGoTime(c *C) {
 	}
 
 }
-<<<<<<< HEAD
```

Comment:
I don't think we should remove these benchmarks?

Reply:
@wjhuang2016 Update. PTAL